CONTRIBUTORS.yaml

```diff
@@ -123,6 +123,11 @@ hmenager:
 
 hrhotz:
     name: Hans-Rudolf Hotz
+    email: hrhotz@googlemail.com
+    twitter: hrhotz
+    gitter: hrhotz
+    orcid: 0000-0002-2799-424X
+    linkedin: Hans-Rudolf Hotz
```

Comment:
mh, should this be: `hans-rudolf-hotz-542b31`

Reply:
where ?

Reply:
...now I understand...will fix it